packages/twenty-server/src/engine/api/graphql/graphql-query-runner/graphql-query-parsers/graphql-query-selected-fields/graphql-selected-fields-relation.parser.ts

```diff
@@ -22,16 +28,34 @@ export class GraphqlQuerySelectedFieldsRelationParser {
   }
 
   parseRelationField(
-    fieldMetadata: FlatFieldMetadata,
+    fieldMetadata:
+      | FlatFieldMetadata<FieldMetadataType.RELATION>
+      | FlatFieldMetadata<FieldMetadataType.MORPH_RELATION>,
     fieldKey: string,
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     fieldValue: any,
     accumulator: GraphqlQuerySelectedFieldsResult,
+    isFromOneToManyRelation?: boolean,
   ): void {
     if (!fieldValue || typeof fieldValue !== 'object') {
       return;
     }
 
+    const isOneToManyRelation =
```

Comment:
this is too late for graphql parsing? in this case should'nt we do it in common api